Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN /bin/bash -c "source /root/.sdkman/bin/sdkman-init.sh && sdk install kotlin"
 ENV PATH="/root/.sdkman/candidates/kotlin/current/bin:${PATH}"
 
 # Python
-RUN add-apt-repository ppa:jonathonf/python-3.6
+RUN add-apt-repository ppa:deadsnakes/ppa
```

Comment:
Could you please elaborate on the change? What's the difference between two repos?

Reply:
Because this repo was closed by owner.

Here is the [log](https://buildkite.com/typeguard/quicktype-pr/builds/1285#fe414035-af96-4c13-9dd2-6815272ebba9) with error